state/pgstatestorage.go

```diff
@@ -112,21 +109,34 @@ func (p *PostgresStorage) getExecQuerier(dbTx pgx.Tx) execQuerier {
 // Reset resets the state to a block for the given DB tx
 func (p *PostgresStorage) Reset(ctx context.Context, blockNumber uint64, dbTx pgx.Tx) error {
 	e := p.getExecQuerier(dbTx)
+	const resetForcedBatchSQL = `
+		UPDATE state.forced_batch 
+		SET batch_num = null
+		FROM state.virtual_batch 
+		INNER JOIN state.block ON state.block.block_num = state.virtual_batch.block_num
```

Comment:
why we need the `inner join` with block table? `virtual_batch` has already `block_num` ???